ksql-functional-tests/src/test/java/io/confluent/ksql/test/tools/TestCaseTest.java

```diff
@@ -124,12 +124,13 @@ public void shouldValidateOutputCorrectly() {
     // Given:
     final TopologyTestDriverContainer topologyTestDriverContainer = getSampleTopologyTestDriverContainer();
     when(topologyTestDriver.readOutput(any(), any(), any()))
-        .thenReturn(new ProducerRecord<>("bar_kafka", 1, 123456789L, "k1", "v1, v2"));
-
+        .thenReturn(new ProducerRecord<>("bar_kafka", 1, 123456789L, "k1", "v1, v2"))
+        .thenReturn(null);
```

Comment:
isn't this the default behaviour?

Reply:
No, default behaviour will return repeatedly return the last value in `thenReturn`